documentation/source/user_section/command-line/sct_deepseg.rst

```diff
@@ -110,11 +113,11 @@ You can replace "``seg_sc_lesion_t2w_sci``" with any of the task names in the ta
    :widths: 25 25 25
 
    * - |seg_sc_lesion_t2w_sci| ``seg_sc_lesion_t2w_sci``
-     - |seg_sc_ms_lesion_stir_psir| ``seg_sc_ms_lesion_stir_psir``
+     - |seg_ms_lesion| ``seg_ms_lesion``
      - |seg_ms_lesion_mp2rage| ``seg_ms_lesion_mp2rage``
-   * - |seg_tumor_edema_cavity_t1_t2| ``seg_tumor_edema_cavity_t1_t2``
+   * - |seg_sc_ms_lesion_stir_psir| ``seg_sc_ms_lesion_stir_psir``
```

Comment:
With @valosekj we were wondering whether removing this model from the model gallery since we don't want people to use it anymore. What are your thoughts?

Reply:
Good idea !